content/documentation/glossary/index.adoc

```diff
@@ -0,0 +1,207 @@
+---
+title: "Glossary"
+date: 2018-06-20T19:04:38+02:00
+draft: false
+type: "documentation"
+---
+
+= Kiali Overview
+:sectnums:
+:toc: left
+toc::[]
+:toc-title: Terminology Content
+:keywords: Kiali Documentation
+:icons: font
+:imagesdir: /images/documentation/terminology/
+
+== Application
+
+It's a logical group of <<Workload>> defined by the <<Label App>>.
+
+=== Application Name
+
+It's the name of the <<Application>> deployed in your environment. This name is provided by the <<App Label>> on the <<Workload>>
+
+
+== Deployment
+
+A deployment is a replication controller based on a user defined template called a <<Deployment>> configuration. Deployments are created manually or in response to triggered events.
```

Comment:
maybe it's worth to link to k8s/okd documentation for entities that come from there?
note - RC is an outdated term that is no longer used by k8s (it is also stated in their docs)

Reply:
I am going to check the documentation thanks